src/layouts/rails-component-layout.tsx

```diff
@@ -29,6 +29,7 @@ export const query = graphql`
           reactId
           railsIds
           figmaId
+          cssId
```

Comment:
I'm not sure what I'm missing but the CSS tab is not appearing if you're current page is the React, Rails, or Figma tab. The CSS tab only shows up if you're on the current tab, or if you're looking at the Overview

Reply:
you might need to add the CSS `Undelinenav.Link` to the the Rails/React and Figma layouts like you did here for Overview:

https://github.com/primer/design/pull/551/files#diff-c4827c6a111be693621e9ce1a0056a1e714d324742542bb266cdae9045a52679R40-R44